python/pacemaker/_cts/environment.py

```diff
@@ -0,0 +1,645 @@
+        grp4.add_argument("--notification-agent",
+                          metavar="PATH",
+                          default="/var/lib/pacemaker/notify.sh",
+                          help="Script to configure for Pacemaker alerts")
+        grp4.add_argument("--notification-recipient",
+                          metavar="R",
+                          default="/var/lib/pacemaker/notify.log",
+                          help="Recipient to pass to alert script")
+        grp4.add_argument("--oprofile",
+                          metavar="NODES",
+                          help="List of cluster nodes to run oprofile on")
+        grp4.add_argument("--outputfile",
+                          metavar="PATH",
+                          help="Location to write logs to")
+        grp4.add_argument("--qarsh",
+                          action="store_true",
+                          help="Use QARSH to access nodes instead of SSH")
+        grp4.add_argument("--schema",
+                          metavar="SCHEMA",
+                          default="pacemaker-3.0",
+                          help="Create a CIB conforming to the given schema")
+        grp4.add_argument("--seed",
+                          metavar="SEED",
+                          help="Use the given string as the random number seed")
+        grp4.add_argument("--set",
+                          action="append",
+                          metavar="ARG",
+                          default=[],
+                          help="Set key=value pairs (can be specified multiple times)")
+        grp4.add_argument("--stonith-args",
+                          metavar="ARGS",
+                          default="hostlist=all,livedangerously=yes",
+                          help="")
+        grp4.add_argument("--stonith-type",
+                          metavar="TYPE",
+                          default="external/ssh",
+                          help="")
+        grp4.add_argument("--trunc",
+                          action="store_true", dest="truncate",
+                          help="Truncate log file before starting")
+        grp4.add_argument("--valgrind-procs",
+                          metavar="PROCS",
+                          default="pacemaker-attrd pacemaker-based pacemaker-controld pacemaker-execd pacemaker-fenced pacemaker-schedulerd",
+                          help="Run valgrind against the given space-separated list of processes")
+        grp4.add_argument("--valgrind-tests",
+                          action="store_true",
+                          help="Include tests using valgrind")
+        grp4.add_argument("--warn-inactive",
+                          action="store_true",
+                          help="Warn if a resource is assigned to an inactive node")
+
+        parser.add_argument("iterations",
+                            type=int,
+                            help="Number of tests to run")
```

Comment:
Shouldn't this be optional and default to 1? At least that was the old behavior which was more intuitive.

Reply:
Ah, that previously happened in CTSlab.py -- yes it would be better to do it here, we can do that for rc2